PubHub/PubHub.Common/Services/HttpClientService.cs

```diff
@@ -1,12 +1,14 @@
 ﻿using Polly;
 using PubHub.Common.Models.Authentication;
 using System.Diagnostics;
+using System.Net.Mime;
 
 namespace PubHub.Common.Services
 {
     public class HttpClientService : IHttpClientService
     {
         private const int RETRY_COUNT = 5;
+        private const string BEARER_KEY = "Bearer";
```

Comment:
Is this used here?

Reply:
It was not, apparently 🤔 
I've fixed it now, though